vhdbuilder/packer/components.json

```diff
@@ -274,7 +274,8 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:*",
       "versions": [
-        "v2.2.0"
+        "v2.2.0",
+        "v2.3.0"
```

Comment:
also need to add it in file `vhdbuilder/packer/generate-windows-vhd-configuration.ps1`

Reply:
fixed